test/Market.t.sol

```diff
@@ -227,13 +225,14 @@ contract Market_replace is MarketReplaceTestHelper {
     });
     uint256 amount = _market.calculateCheckoutTotal(_amountToReplace);
     vm.startPrank(_namedAccounts.admin);
-    _removal.release(_removalIds[0], _originalCertificateAmount);
-    assertEq(_certificate.getNrtDeficit(), _originalCertificateAmount);
     _bpNori.deposit(_namedAccounts.admin, abi.encode(amount));
     _bpNori.approve(address(_market), amount);
+    _removal.release(_removalIds[0], _originalCertificateAmount);
+    vm.stopPrank();
```

Comment:
To reproduce the original issue, simply remove this line of code and you'll get a panick from revm

```
The application panicked (crashed).                                                                                                                        
Message:  called `Option::unwrap()` on a `None` value                        
Location: ~/.cargo/git/checkouts/revm-bf744d8ffabcbad9/bc4d203/crates/revm/src/evm_impl.rs:267  
```